### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,

- Create React app
- Linters
- GitHub flow
- Descriptive README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,170 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./assets/image/book-icon.png" alt="logo" width="140"  height="auto" />
```
Comment (wayungi) [inline README.md]: - [ ] Your logo is not displaying, please make sure the image is upload to GitHub and that it is properly linked  👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Aqurds , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All Linter checks are passing ✔️ 
- [ ] CRA used to create the app ✔

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please note the error when your application loads. Make sure your Linters are well configured and that you are using the right Linters for React
err


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Aqurds) [reply README.md]: @wayungi Thanks for your review :) would you please help me to solve the error when app runs? I checked it and rerun the application on my machine. I didn't see any error. The app is running in the browser perfectly. If you like, then I can send you a video on it. Thanks again :)

**Review by muneebulrehman — APPROVED** [review]
Hi @Aqurds , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**